src/components/date/date.spec.js

```diff
@@ -91,11 +91,14 @@ describe("Date", () => {
   let wrapper;
   let onFocusFn;
 
-  // eslint-disable-next-line react/prop-types
   const MockComponent = ({
+    // eslint-disable-next-line react/prop-types
```

Comment:
question (non-blocking): Likewise for spec.js files, maybe we should just turn the rule off altogether?

Reply:
I agree, if we plan to release this work once all our components are migrated to TypeScript, there will be little point of keeping this rule.